libs/ui/src/hooks/auth/auth_helpers.ts

```diff
@@ -348,6 +348,15 @@ export function isPollWorkerAuth(
 
 export function isCardlessVoterAuth(
   auth: InsertedSmartcardAuth.Auth
-): auth is InsertedSmartcardAuth.CardlessVoterLoggedIn {
+): auth is InsertedSmartcardAuth.CardlessVoterLoggedIn;
+export function isCardlessVoterAuth(
+  auth: InsertedSmartCardAuth.AuthStatus
+): auth is InsertedSmartCardAuth.CardlessVoterLoggedIn;
+export function isCardlessVoterAuth(
+  auth: InsertedSmartcardAuth.Auth | InsertedSmartCardAuth.AuthStatus
+): auth is InsertedSmartCardAuth.CardlessVoterLoggedIn;
+export function isCardlessVoterAuth(
+  auth: InsertedSmartcardAuth.Auth | InsertedSmartCardAuth.AuthStatus
+): boolean {
```

Comment:
After this PR is merged, I'll clean up all the old, now unused auth code, including the old types!